sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -925,6 +925,26 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     }
   }
 
+  test("show table extended ... partition") {
```

Comment:
Then, you just need to improve the function `getNormalizedResult` in SQLQueryTestSuite to mask it.